src/Admin/AbstractAdmin.php

```diff
@@ -1122,33 +1130,58 @@ public function generateMenuUrl($name, array $parameters = [], $absolute = Routi
         return $this->routeGenerator->generateMenuUrl($this, $name, $parameters, $absolute);
     }
 
+    public function setTemplateRegistry(MutableTemplateRegistryInterface $templateRegistry)
+    {
+        $this->templateRegistry = $templateRegistry;
+    }
+
+    /**
+     * @deprecated Use TemplateRegistry services instead
```

Comment:
If this is really deprecated, then you should trigger a deprecation. See the CONTRIBUTING.md guide for more details.

Reply:
I'll remove the deprecation here until the admin specific configuration is implemented (see "next steps"). Until then, this is the only way to set the templates.

Reply:
Ah ok then.